CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## [Unreleased]
```

Comment:
The unreleased tag should be removed such that the first one that shows up when you go to the release on github is `v0.5.0`. We'll add it again afterwards.

Reply:
fxied